src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_bc_system_capability_updated_notification_from_hmi.cc

```diff
@@ -104,10 +105,21 @@ void OnBCSystemCapabilityUpdatedNotificationFromHMI::Run() {
   (*message_)[strings::params][strings::function_id] = static_cast<int32_t>(
       mobile_apis::FunctionID::OnSystemCapabilityUpdatedID);
 
+  if (!(*message_)[strings::msg_params].keyExists(strings::system_capability)) {
```

Comment:
@IGapchuk `systemCapability` - mandatory parameter, so I think this check is redundant

Reply:
@LitvinenkoIra removed in 7bd8a8207b1d42cc9b6c6c94ee1c0b5028e13924